### PR TITLE
fix automation-anywhere theme not shown on first login

### PR DESCRIPTION
**Pull request by fungairino**
This fixes an edge case where the partner theme wasn't loaded
on first login - fixing a failing rainforest qa test

## What does this PR do?

- Closes/Fixes/Part of #123
- _Are there other changes (e.g., refactoring) included that aren’t from the ticket?_

## Reviewer Tips

- _What order should the reviewer review the files in?_
- _Are there any implementation approaches you want feedback on?_

## Discussion

- _Were there multiple approaches you were deciding between?_

## Demo

- _Paste a screenshot or demo video here_

## Future Work

- _Work for the issue/ticket that will be in a follow-up PR_

## Team Coordination

_Leave all that are relevant and check off as completed_

- [ ] This PR requires security review
- [ ] This PR introduces a new library: double check it's MIT/Apache2/permissively licensed
- [ ] This PR requires a node/npm version update: let the team know on #engineering
- [ ] This PR requires a documentation change (link to old docs)
- [ ] This PR requires a tutorial update (link to old tutorials)
- [ ] This PR requires a feature flag
- [ ] This PR requires a environment variable change

## Checklist

- [ ] Add tests
- [ ] New files added to `src/tsconfig.strictNullChecks.json` (if possible)
- [ ] Designate a primary reviewer

### CODE REVIEW

File: src/store/settings/settingsSlice.ts [hunk 1]
```diff
@@ -86,6 +87,8 @@ const settingsSlice = createSlice({
       { payload: { partnerId } }: { payload: { partnerId: string } },
     ) {
       state.partnerId = partnerId;
+      // `activateTheme` in background script triggers updating themeStorage
+      void activateTheme();
```
Comment (BLoe) [inline src/store/settings/settingsSlice.ts:91]: We're trying to avoid `void` calls inside redux reducers, and have a DX initiative to remove them. Do you mind converting this to an async thunk? Or calling `activateTheme()` inline at the call sites where this action is dispatched?

Comment (fungairino) [reply src/store/settings/settingsSlice.ts:91]: Sure thing

**Review by twschiller — APPROVED** [review]
